css/app/eventdialog.css

```diff
@@ -120,6 +120,10 @@
 	width: 60px;
 }
 
+.events .events--time[disabled] {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events

click doesn't work on disabled inputs. With `pointer-events:none` the clicks on the input do straight to the parent div